Makefile

```diff
@@ -261,7 +261,15 @@ bundle-push: ## Push the bundle image.
 
 .PHONY: deploy-bundle
 deploy-bundle: operator-sdk bundle-build bundle-push
-	$(OPERATOR_SDK) run bundle $(BUNDLE_IMG)
+	$(OPERATOR_SDK) run bundle $(BUNDLE_IMG)  -n $(OPERATOR_NAMESPACE)
+
+.PHONY:	create-namespace
+create-namespace: ## Create namespace
```

Comment:
Why do we need this target?

Reply:
Initially I was thinking that we need to create a separate namespace for upgrade testing but as we discussed earlier we are going to use existing namespace `openshift-operators`. I'll remote it.